From 15ec33d5109b81e9e165f08fb061c76d3ca8f218 Mon Sep 17 00:00:00 2001
From: Jarkko Nikula <jhnikula@gmail.com>
Date: Mon, 18 Apr 2011 19:21:32 +0000
Subject: [PATCH 0325/1092] omap: Fix board-nokia770.c and board-n8x0 cbus
 build breakage

Fix a build breakage in CBUS initialization code in Nokia 770 and N8x0
boards that appeared after commit 0c6f8a8 "genirq: Remove compat code".

Signed-off-by: Jarkko Nikula <jhnikula@gmail.com>
Signed-off-by: Tony Lindgren <tony@atomide.com>
---
 arch/arm/mach-omap1/board-nokia770.c | 2 +-
 arch/arm/mach-omap2/board-n8x0.c     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-omap1/board-nokia770.c b/arch/arm/mach-omap1/board-nokia770.c
index 1ba0ffd..6bc6f33 100644
--- a/arch/arm/mach-omap1/board-nokia770.c
+++ b/arch/arm/mach-omap1/board-nokia770.c
@@ -161,7 +161,7 @@ static void __init nokia770_cbus_init(void)
 		return;
 	}
 
-	set_irq_type(gpio_to_irq(62), IRQ_TYPE_EDGE_RISING);
+	irq_set_irq_type(gpio_to_irq(62), IRQ_TYPE_EDGE_RISING);
 	retu_resource[0].start = gpio_to_irq(62);
 	platform_device_register(&retu_device);
 
diff --git a/arch/arm/mach-omap2/board-n8x0.c b/arch/arm/mach-omap2/board-n8x0.c
index 28f9a60..5a2a073 100644
--- a/arch/arm/mach-omap2/board-n8x0.c
+++ b/arch/arm/mach-omap2/board-n8x0.c
@@ -272,7 +272,7 @@ static void __init n8x0_cbus_init(void)
 		return;
 	}
 
-	set_irq_type(gpio_to_irq(108), IRQ_TYPE_EDGE_RISING);
+	irq_set_irq_type(gpio_to_irq(108), IRQ_TYPE_EDGE_RISING);
 	retu_resource[0].start = gpio_to_irq(108);
 	platform_device_register(&retu_device);
 
-- 
1.7.11.2

